django/utils/text.py

```diff
@@ -327,9 +348,15 @@ def read(self):
 
 
 # Like compress_string, but for iterators of strings.
-def compress_sequence(sequence):
+def compress_sequence(sequence, max_random_bytes=0):
     buf = StreamingBuffer()
-    with GzipFile(mode="wb", compresslevel=6, fileobj=buf, mtime=0) as zfile:
+    with GzipFile(
+        mode="wb",
+        filename=_get_random_fname(max_random_bytes) if max_random_bytes else None,
```

Comment:
_This should be fine if provided with a `bytes` filename... At least if `typeshed` is correct..._